src/routes/search/components/SearchFeedback.js

```diff
@@ -49,7 +49,7 @@ export class SearchFeedback extends React.Component<PropsType, StateType> {
   openFeedbackBox = () => {
     const {location, query} = this.props
     const {city, language} = location.payload
-    FeedbackEndpoint.postData({
+    feedback.fetchData({
```

Comment:
imo fetchData sounds like we're expecting to get data from the backend... maybe .request() would be a better name now?

Reply:
request sounds good :+1: